platform/core/README.md

```diff
@@ -9,13 +9,13 @@ Logic shared across polyPod platforms - ios, android, wasm.
 
 ## Building
 
--   When setting up the project for the first time or switching between branches, the flatbuffers models have to be generated:
+- Setup flatbuffers:
+          ./flatbuffers_shared/setup_flatbuffers.sh
```

Comment:
nit-nit
```suggestion
- Set up flatbuffers:
          sh /flatbuffers_shared/setup_flatbuffers.sh
```

Reply:
From my tests, `sh` does not work on Windows(with my very basic setup); also using `sh` on Linux for some reason makes `$OSType` in `setup_flatbuffers.sh` to be empty...

Reply:
oh alright then 👍🏽 
(it was mainly a nit-nit for the set up instead setup :P (got the same comment from @fhd this morning 🤓 😆 ))

Reply:
aah, right. Idk why I wrote `Set up`...